go/rpc/client.go

```diff
@@ -8,12 +8,12 @@ import (
 )
 
 const (
```

Comment:
I renamed these constants so that methods that relate to rollups are called "rollupX", not "blockX" (the actual method names cannot change, because they're used by stuff like Remix).

Reply:
Good idea imo